### ISSUE TITLE
Add option to disable kind projector

### ISSUE THREAD

**Issue by kubukoz** [typelevel/sbt-typelevel#375]
Hi! It'd be nice to have an easy way to disable kind-projector, e.g. for builds of other compiler plugins (so that they can be built for a Scala version KP doesn't support).

**Comment by armanbilge** [comment]
Would be good to have a setting for `better-monadic-for` as well.

**Comment by satorg** [comment]
Hmm.. If I'm not mistaken, it became a compiler option in Scala 3, didn't it?

**Comment by kubukoz** [comment]
Yes... but we're still getting new 2.x versions.

**Comment by satorg** [comment]
Right.. I'm just thinking about integration with `scalac-options` and how to address such differences..

**Comment by armanbilge** [comment]
> Right.. I'm just thinking about integration with `scalac-options` and how to address such differences..

Right, that's gnarly. That's the kind of thing that has to be resolved in sbt-typeleve-settings I think, since scalac-options can't really do anything in this area for Scala 2.
